### DISCUSSION TITLE
Does the "Mod select hotkey style" change anything since the change in which you automatically focus the Search input tile in mod select?

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#26265] [Ideas] [answered]
I'd much rather prefer using classic hotkeys for classic mods, and I don't know how to do that now

![image](https://github.com/ppy/osu/assets/82095491/f742cef5-3406-4c1e-b6ea-ef159ba54df9)

**Comment by Gabixel** [comment, marked as answer]
There's a new option enabled by default, search for "focus". You can disable it, or press TAB to unfocus the mod search before pressing the hotkeys
![image](https://github.com/ppy/osu/assets/43073074/24b0d348-60cc-46a4-ab08-d2e314e6f47a)

**Reply by Skubers** [reply]
oioi, I'm actually acoustic that I didn't notice that. Thanks a lot

Edit: AND IT'S RIGHT UNDER THE HOTKEY STYLE SELECTION BOX, I'M DEAD
